### PR TITLE
removed unnecessary functions

### PR DESCRIPTION
**Pull request by BartMoroz**

### CODE REVIEW

File: wpwithinpy/launcher.py [hunk 1]
```diff
@@ -33,31 +33,18 @@ def launch(self, cfg, path, flags):
...
-                return process
-            elif(hostOS == "win32"):
-                process = self.launchWindows(path, flags)
-                return process
-            elif (hostOS == "windows"):
-                process = self.launchWindows(path, flags)
-                return process
+            fnull = open(os.devnull, 'w')
+            if os.environ.get("WPW_HOME") is not None:
+                cmd = [os.environ[
```
Comment (maciej-wakula-mobica) [inline wpwithinpy/launcher.py]: Strange line break - please, fix it.

File: wpwithinpy/launcher.py [hunk 2]
```diff
@@ -33,31 +33,18 @@ def launch(self, cfg, path, flags):
...
+                           "WPW_HOME"] + '/bin/rpc-agent-' + platform.system().lower() + '-' + self.detectHostArchitecture()]
```
Comment (maciej-wakula-mobica) [inline wpwithinpy/launcher.py]: will slash work on windows?

File: wpwithinpy/launcher.py [hunk 3]
```diff
@@ -33,31 +33,18 @@ def launch(self, cfg, path, flags):
...
             else:
-                logging.debug("Unable to launch binary on host architecture (Unsupported by launcher)(Host=" + str(hostOS) + ")")
+                cmd = [path + '/wpwithinpy/iot-core-component/bin/rpc-agent-' + self.detectHostOS() + '-' + self.detectHostArchitecture()]
```
Comment (maciej-wakula-mobica) [inline wpwithinpy/launcher.py]: What is the OS is not supported and binary is not there?

File: wpwithinpy/launcher.py [hunk 4]
```diff
@@ -33,31 +33,17 @@ def launch(self, cfg, path, flags):
...
+            cmd.extend(flags)
+            proc = subprocess.Popen(cmd, stdin=None, stdout=fnull, stderr=subprocess.STDOUT)
+            return proc
         else:
             logging.debug("Invalid OS/Architecture combination detected")
 
-    def detectHostOS(self):
```
Comment (maciej-wakula-mobica) [inline wpwithinpy/launcher.py]: Since it is removed here - you cannot use it on line 40

Comment (BartMoroz) [reply wpwithinpy/launcher.py]: fixd

**Pull request merged**